projects/valtimo/zgw/src/lib/modules/documenten-api/components/documenten-api-metadata-modal/documenten-api-metadata-modal.component.html

```diff
@@ -187,37 +189,33 @@ <h3 cdsModalHeaderHeading>
               {{ option.translationKey | translate }}
             </cds-radio>
           </cds-radio-group>
-
-          @if (supportsTrefwoorden) {
-            <cds-text-label>
-              {{ 'document.tags' | translate }}
-              <cds-tooltip
-                [isOpen]="false"
-                [description]="'document.tagsTooltip' | translate"
-                [align]="'bottom-left'"
-              >
-                <i class="mdi mdi-help-circle-outline"></i>
-              </cds-tooltip>
-            </cds-text-label>
-            <div class="documenten-metada-form-input">
-              <cds-combo-box
-                [type]="'multi'"
-                [placeholder]="'document.tagsPlaceholder' | translate"
-                [disabled]="obs.disabled"
-                (selected)="tagsSelected($event)"
+          <cds-text-label>
```

Comment:
The code to optinally show trefwoorden input `@if (supportsTrefwoorden) {` should not be removed, but included